client/signup/utils.js

```diff
@@ -170,10 +170,14 @@ function getThemeForSiteGoals( siteGoals ) {
 function getSiteTypeForSiteGoals( siteGoals ) {
 	const siteGoalsValue = siteGoals.split( ',' );
 
-	if ( siteGoalsValue.indexOf( 'sell' ) !== -1 ) {
+	if ( siteGoals === 'sell' ) {
```

Comment:
Down the road it might be difficult to remember why we handle the singular sell goal differently here. Maybe add a comment before this line explaining the rationale.

Reply:
Good call, thanks @allendav